### ISSUE TITLE
DOM elements don't support remove method in IE 10.

### ISSUE THREAD

**Issue by CoderK** [humanmade/react-oembed-container#5]
In IE 10, DOM Element doesn't support remove method.
(https://stackoverflow.com/a/37477096/3349945)

So the code below will not work properly in IE 10.

```javascript
      if (!document.querySelector('body > #fb-root')) {
        // There may be multiple #fb-root elements in a post's content: remove them
        // all in favor of a new body-level div.
        [...document.querySelectorAll('#fb-root')].forEach(n => n.remove());

        // Prepare and create the fb-root element. We only need one.
        const fbDiv = document.createElement('div');
        fbDiv.id = 'fb-root';
        document.body.prepend(fbDiv);
      }
```
https://github.com/humanmade/react-oembed-container/blob/master/src/embeds.js#L18

I think it would be nice to change this in the following way for a wider browser support. What do you think?

**Comment by CoderK** [comment]
Oops. The same goes for prepend method.
https://github.com/humanmade/react-oembed-container/blob/master/src/embeds.js#L23

**Comment by saifbechan** [comment]
I added a PR for this: https://github.com/humanmade/react-oembed-container/pull/7/files

**Comment by kadamwhite** [comment]
Merged #7, releasing this as a beta tag shortly and should have a formal release out soon after.
